nav2_dwb_controller/dwb_controller/include/dwb_controller/dwb_controller.hpp

```diff
@@ -26,6 +26,7 @@
 #include "nav2_util/lifecycle_node.hpp"
 #include "nav2_msgs/action/follow_path.hpp"
 #include "nav2_util/simple_action_server.hpp"
+#include "dwb_controller/progress_checker.hpp"
```

Comment:
Not a big deal since we're not consistent with this, but since you're only using a reference you could declare the ProgressChecker class instead of including the header, then include the header in the implementation file.. This becomes more important on large projects when one is trying to keep compile time from getting out of hand.

Reply:
Agree, I'd like to start doing that. Will make a change.